tasks/task2/draw2.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+"""
+This script reads in the given file, and generates a graph of the words in it
+usage: ./draw.py ocroutput.day
+"""
+
+import csv
+import logging
+import sys
+
+import matplotlib.patches as patches
+import matplotlib.pyplot as plt
+
+import pandas as pd
+import statistics
+
+logging.basicConfig(level=logging.INFO)
+
+def getColumnDelimiters(fName):

```

Comment:
There is some good documentation here, but could use more parameter information. There are also some opportunities for splitting out  smaller functions to increase modularity.